docs/framework/deployment/index.md

```diff
@@ -28,7 +28,7 @@ ms.author: "mairaw"
 manager: "wpickett"
 ---
 # Deploying the .NET Framework and Applications
-This article helps you get started deploying the .NET Framework with your application. Most of the information is intended for developers, OEMs, and enterprise administrators. Users who want to install the .NET Framework on their computers should read [Installing the .NET Framework 4.5](../../../docs/framework/getting-started/install-the-net-framework.md).  
+This article helps you get started deploying the .NET Framework with your application. Most of the information is intended for developers, OEMs, and enterprise administrators. Users who want to install the .NET Framework on their computers should read [Installing the .NET Framework](~/docs/framework/install/guide-for-developers.md).  
```

Comment:
This is the wrong link -- we want a user installation topic, not a developer installation topic. The link should go to the top-level page that Rich wrote, [Installing the .NET Framework on Windows](../install/index.md).